modules/map/hdmap/adapter/xml_parser/junctions_xml_parser.cc

```diff
@@ -48,16 +49,36 @@ Status JunctionsXmlParser::Parse(const tinyxml2::XMLElement& xml_node,
       return Status(apollo::common::ErrorCode::HDMAP_DATA_ERROR, err_msg);
     }
 
+    std::string junction_type;
+    checker =
+      UtilXmlParser::QueryStringAttribute(
+        *junction_node, "type", &junction_type);
```

Comment:
Please try `scripts/clang-format.sh` to correct obvious style and layout issues.